collections/_article/release-management-4-1.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Release Management Post Godot 4.0"
+excerpt: "We plan on releasing Godot 4.1 at the end of June after three months of feature merging and one month of bug fixing."
+categories: ["news"]
+author: Clay John
+image: /storage/app/uploads/public/638/55e/dad/63855edad2371754303509.png
+date: 2023-04-19 14:00:00
+---
+
+Now that [Godot 4.0 is out](https://godotengine.org/article/godot-4-0-sets-sail/) and we have all returned from [GDC](https://godotengine.org/article/gdc-2023-retrospective/) we can finally take the time to solidify our release management plan for the 4.x releases (starting with 4.1). In our [earlier blog post](https://godotengine.org/article/release-management-4-0-and-beyond/) we mentioned that we would like to shift to smaller, more regular releases. Our hope is that we can avoid some of the crunch that comes with trying to squeeze big features into the next release and the resulting long alpha/beta periods that result from cramming in too many big features into a release. 
+
+The current plan is to organize releases into two phases: the feature merging phase and the bug fixing phase. 
```

Comment:
Next sentence starts with *the current plan* too in a different context. I'm too tired to come up with an alternative, and it doesn't matter much, just wanted to note it :P

Reply:
I've expanded upon it in my suggestions above. Should address your concern, perhaps? :)

Reply:
I'll rephrase one of them!